spec/ims/lti/models/messages/message_spec.rb

```diff
@@ -2,50 +2,53 @@
 
 module IMS::LTI::Models::Messages
   describe Message do
+
+    let(:message) {Message.new}
```

Comment:
I updated let to a named subject, the problem I see with using "described_class" is that it creates a layer of abstraction  around what is going on. using `Message.new` is intention revealing, and less space